packages/core/contracts/oracle/implementation/OptimisticOracle.sol

```diff
@@ -127,7 +127,8 @@ contract OptimisticOracle is OptimisticOracleInterface, Testable, Lockable {
 
     // This is effectively the extra ancillary data to add ",ooRequester:0000000000000000000000000000000000000000".
     uint256 private constant MAX_ADDED_ANCILLARY_DATA = 53;
-    uint256 public constant OO_ANCILLARY_DATA_LIMIT = ancillaryBytesLimit - 53;
+    uint256 public constant OO_ANCILLARY_DATA_LIMIT = ancillaryBytesLimit - MAX_ADDED_ANCILLARY_DATA;
+    int256 public constant TOO_EARLY_RESPONSE = type(int256).min;
```

Comment:
would be cool to refactor this in a way that we can re-use this magic number throughout some of the other contracts in this package such as here https://github.com/UMAprotocol/protocol/blob/faa54932d9a5bb4dc46bc5337def8707ff44159d/packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol#L348

Reply:
Agree! But let's do this in a follow-up just to keep these fixes focused and simple.